src/currencyWallets/currency.test.js

```diff
@@ -137,6 +144,28 @@ describe('currency wallets', function () {
     })
   })
 
+  it('getMaxSpendable', async function () {
+    const store = makeFakeCurrencyStore()
+    store.dispatch({ type: 'SET_BALANCE', payload: 50 })
+
+    const wallet = await makeFakeCurrencyWallet(store)
+    const maxSpendable = await wallet.getMaxSpendable({
+      currencyCode: 'TEST',
+      spendTargets: [{}]
+    })
+
+    assert.isFulfilled(
```

Comment:
These don't actually pass the flow type checker. I'm not sure how to fix this - `chai-as-promised` seems to not have flow typings. I suppose we could just do `assert.equals(await wallet.getMaxSpendable(...), someAmount)`, bypassing the whole `chai-as-promised` library. If you can find flow typings for it instead, that would be nice too.

Reply:
yes, but on the next line i want to check that promise is rejected and i cant do this without chai plugins